src/rust/examples/slapper/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Generated build files
+/target
+Cargo.lock
```

Comment:
`Cargo.lock` can be removed from the `.gitignore`, as it's an executable program and the file is actually commited either way.